changelog/4137.featuredownloadsamplelog.rst

```diff
@@ -0,0 +1 @@
+Added logs for failure while downloading sample data.
```

Comment:
```suggestion
Added log messages when the sample data fails to download.
```
This file should also be named `4137.trival.rst`

Reply:
Done.